roles/prometheus/templates/prometheus.service.j2

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Prometheus
+Wants=network-online.target
+After=network-online.target
+[Service]
+User=prometheus
```

Comment:
This needs to follow the `prometheus_user` and `prometheus_group` role variables.